core/src/main/java/io/lionweb/lioncore/java/language/Classifier.java

```diff
@@ -50,6 +50,7 @@ public Classifier(@Nullable Language language, @Nullable String name) {
 
   public @Nonnull List<Classifier<?>> allAncestors() {
     List<Classifier<?>> ancestors = new ArrayList<>();
+    // TODO this implementation seems to be wrong
```

Comment:
This is indeed very wrong. But probably we expect this to be overriden. Maybe this should just be abstract

Reply:
Implemented a fixed generic version